modules/Bio/EnsEMBL/Compara/PipeConfig/HomologyAnnotation_conf.pm

```diff
@@ -81,9 +88,19 @@ sub default_options {
         'decision_capacity'        => 150,
         'hc_priority'              => -10,
 
-        'num_sequences_per_blast_job'   => 200,
-        'blast_params'              => '--max-hsps 1 --threads 4 -b1 -c1 --sensitive',
-        'evalue_limit'              => '1e-6',
+        'num_sequences_per_blast_job' => 200,
+        'blast_params'                => '--max-hsps 1 --threads 4 -b1 -c1 --sensitive',
+        'evalue_limit'                => '1e-6',
+
+        'reference_list' => {
+            'vertebrates' => ['homo_sapiens', 'danio_rerio', 'gallus_gallus', 'mus_musculus'],
+            'metazoa'     => [ ],
+            'plants'      => [ ],
+            'fungi'       => [ ],
+            'bacteria'    => [ ],
+            'protists'    => [ ],
+            'all'         => ['homo_sapiens', 'danio_rerio', 'gallus_gallus', 'mus_musculus'],
+        },
```

Comment:
This is currently hardcoded, but for [reference side searches back to query](https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-4077) - this will be a `taxon_list` that will refer back to the `reference_master_db`, collect all the `genome_dbs` there and pass those through (this will also be addressed in the prevention of loading reference sequences in `AddRapidSpecies.pm`. This will all be addressed in another PR for [ENSCOMPARASW-4077](https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-4077) and [ENSCOMPARASW-3768](https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-3768)

Reply:
Yes, that is fine, I just meant that for now this is here to make current pipeline work, for next PR it will be gone :slightly_smiling_face: